### PR TITLE
adding SemPend wrapper functionality

### PR DESCRIPTION
**Pull request by connorl309**

### CODE REVIEW

**Review by ray4477 — APPROVED** [review]
looks good ;)

File: Tasks/Src/RTOS_BPS.c [hunk 1]
```diff
@@ -4,9 +4,20 @@ RTOS_BPS library includes all the wrapper functions for a functioning RTOS - nee
...
 #include "RTOS_BPS.h"
 
-void RTOS_BPS_SemPend(void) {
+/**
+ * @brief Pends a BPS_OS_Semaphore.
+ * @param *sem - pointer to a sempaphore to pend
+ * @param tick - time in clock ticks to timeout for
+ * @param opt - pend option
+ * @return the semaphore count, or 0 if not available
+ */
```
Comment (manthanand) [inline Tasks/Src/RTOS_BPS.c:13]: Change the comment headers to match the input parameters.

Comment (connorl309) [reply Tasks/Src/RTOS_BPS.c:13]: wdym, they do match the input parameters unless i am just blind
image

Comment (manthanand) [reply Tasks/Src/RTOS_BPS.c:13]: No, I'm blind

**Pull request merged**